### PR TITLE
Changes to various metadata orgins and dashboard materializations

### PR DESCRIPTION
**Pull request by Sami1309**

### CODE REVIEW

File: dashboard/src/components/entitypage/elements/PrometheusGraph.js [hunk 1]
```diff
@@ -69,17 +89,19 @@ const PrometheusGraph = ({
 
         plugins: {
           "datasource-prometheus": {
-            prometheus: {
-              endpoint: "http://localhost:9090",
-            },
-            query: `${query}${add_labels_string}`,
+            // prometheus: {
```
Comment (simba-git) [inline dashboard/src/components/entitypage/elements/PrometheusGraph.js:92]: Remove comment

File: dashboard/src/components/resource-list/ResourceListView.js [hunk 2]
```diff
@@ -99,7 +99,7 @@ export const ResourceListView = ({
       },
       { title: "Revision", field: "revision" },
       {
-        title: "Version",
+        title: "Variants",
         field: "versions",
```
Comment (simba-git) [inline dashboard/src/components/resource-list/ResourceListView.js:103]: should this still be versions?

**Review by simba-git — COMMENTED** [review]
Minor change first

File: dashboard/src/components/resource-list/ResourceListView.js (hunk 2 above)
Comment (Sami1309) [reply dashboard/src/components/resource-list/ResourceListView.js:103]: it displays variants but retrieves the portions of the json file named "version" for logical purposes. Quicker to change this now than change the name for everything, though will fix that later.

**Pull request merged**